src/main/java/uk/gov/ons/census/casesvc/service/CaseService.java

```diff
@@ -105,24 +105,35 @@ public PayloadDTO saveCaseAndEmitCaseCreatedEvent(Case caze) {
     return saveCaseAndEmitCaseCreatedEvent(caze, null);
   }
 
+  public PayloadDTO saveCaseAndEmitCaseCreatedEvent(Case caze, Metadata metadata) {
```

Comment:
Not asking for this to be changed for this ticket, as it's not the fault of this ticket - but I've got reservations about the size and scope of the CaseService class.  It seems to be doing quiet a lot of sort of related stuff that might, might benefit from refactoring to make it more maintainable

Reply:
I agree that there seems to have been a massive proliferation of overloaded methods, and I don't know why. It makes it quite hard to trace what the heck is going on. On the plus side though, we do generally only have to change stuff in one place, I found while doing this PR